artifact/rules.bzl

```diff
@@ -148,22 +148,43 @@ def artifact_file(name,
         **kwargs
     )
 
-script_template = """\
+script_template_gz = """\
```

Comment:
Let's make this `script_template_targz` to be consistent with naming in our other rules (such as `assemble_targz`)

Reply:
in https://github.com/graknlabs/bazel-distribution/pull/253/commits/07cd24b22dad38db1a949adb58388f514abaa9e0 I've renamed this to `script_template_tar` after its extraction method